startup/gui/ocio.py

```diff
@@ -100,6 +100,7 @@ def __plugSet( plug ) :
 def __displayTransformCreator( name ) :
 
 	result = GafferImage.DisplayTransform()
+	result["channels"].setValue( "*[RGB]" )
```

Comment:
I believe it only applies to RGB channels of arbitrary layers. So. R, G, B, diffuse.R, diffuse.G, diffuse.B would all be affected but Z, A, whatever.Z, whatever.A would not be affected.